backend/services/implementations/classService.ts

```diff
@@ -61,6 +61,7 @@ class ClassService implements IClassService {
       className: newClass.className,
       startDate: newClass.startDate,
       gradeLevel: newClass.gradeLevel,
+      isActive: newClass.isActive,
```

Comment:
I think you will need to update line 50 to have something like 
```
newClass = await MgClass.create({
        ...classObj,
        isActive: true,
        testSessions: [],
        students: [],
      });
```
assuming that classes are always active when first created